### PR TITLE
adds new skill condition MSC_MOBNEARBYGT

### PR DESCRIPTION
**Pull request by datawulf**
* **Server Mode**: Both

* **Description of Pull Request**: 

Adds new monster skill cast condition as on official some monsters trigger skills when specific amount of other monsters are nearby.

### CODE REVIEW

**Comment by mmo4th** [comment]
Offtopic: does rAthena have the skill condition when "monster is receiving ANY SKILL attack"?
rA do have long ranged and melee attack trigger point.

**Comment by Atemo** [comment]
Do you have an example of monster using a skill under MSC_MOBNEARBYGT ?

**Comment by datawulf** [comment]
> Do you have an example of monster using a skill under MSC_MOBNEARBYGT on official?

20262,ILL_MINERAL@AL_TELEPORT,chase,26,1,5000,0,5000,yes,self,mobnearbygt,4,,,,,,,

You can get this info by using the divine-pride API

**Review by Atemo — APPROVED** [review]

**Comment by datawulf** [comment]
> ack"?
> rA do have long ranged and melee attack trigger point.

Yes, just use skillused without an ID
1251,KNIGHT_OF_WINDSTORM@WZ_STORMGUST,chase,89,10,10000,500,5000,no,target,skillused,,,,,,,21,

File: db/import-tmpl/mob_skill_db.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@
 //	afterskill		After mob casts the specified skill.
 //	casttargeted		When a target is in cast range (no condition value).
 //	rudeattacked		When mob is rude attacked (no condition value).
+//	mobnearbygt		When more monsters in range than specified number.
```
Comment (aleos89) [inline db/import-tmpl/mob_skill_db.txt:46]:
```suggestion
//	mobnearbygt		When monsters in range become greater than specified number.
```

File: db/pre-re/mob_skill_db.txt [hunk 2]
```diff
@@ -44,6 +44,7 @@
 //	afterskill		After mob casts the specified skill.
 //	casttargeted		When a target is in cast range (no condition value).
 //	rudeattacked		When mob is rude attacked (no condition value).
+//	mobnearbygt		When more monsters in range than specified number.
```
Comment (aleos89) [inline db/pre-re/mob_skill_db.txt:47]:
```suggestion
//	mobnearbygt		When monsters in range become greater than specified number.
```

File: db/re/mob_skill_db.txt [hunk 3]
```diff
@@ -44,6 +44,7 @@
 //	afterskill		After mob casts the specified skill.
 //	casttargeted		When a target is in cast range (no condition value).
 //	rudeattacked		When mob is rude attacked (no condition value).
+//	mobnearbygt		When more monsters in range than specified number.
```
Comment (aleos89) [inline db/re/mob_skill_db.txt:47]:
```suggestion
//	mobnearbygt		When monsters in range become greater than specified number.
```

**Review by aleos89 — APPROVED** [review]
Just some small documentation fixes.

**Review by Daegaladh — APPROVED** [review]

**Pull request merged**